src/content/pages/pygamezero.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: PyGameZero Workshop
+subtitle: An introduction and fun workshop to teach beginners how to using Python to make games.
+---
+
+# PyGameZero Workshop
+Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this short introduction workshop, we will learn about PyGameZero, a simple framework in Python that allow any coding beginners to create a simple game using Python.
```

Comment:
```suggestion
Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this short workshop, we will learn about PyGameZero - a simple framework in Python that allow coding beginners to create a simple game using Python.
```

Reply:
Maybe add the duration of the workshop?

> In this **X hour-long** workshop, (...)

Or something on these lines :)